packages/atoms/src/components/kiwi-dropdown/kiwi-dropdown.tsx

```diff
@@ -72,6 +77,32 @@ export class KiwiDropdown implements ComponentInterface {
   private handleMenuEnter: () => void = () => (this.hover = true);
   private handleMenuLeave: () => void = () => (this.hover = false);
 
+  componentDidRender(): void {
+    this.adjustPositionAtScreenEdge();
+  }
+
+  /**
+   * Make sure that the dropdown does not stick over the right edge of the body
+   */
+  private adjustPositionAtScreenEdge(): void {
+    if (this.dropdownMenu) {
+      if (this.open) {
+        const rect = this.dropdownMenu.getBoundingClientRect();
+
+        if (rect.right > window.visualViewport.width) {
+          const offset =
+            rect.width > window.visualViewport.width
+              ? Math.floor(rect.left)
+              : Math.ceil(rect.right - window.visualViewport.width);
+
+          this.dropdownMenu.style.translate = `-${offset}px`;
```

Comment:
Funktioniert das so bei dir?
Bei mir klappt es leider nicht und ich meine es müsste `transform: translate(<xOffset>)` heißen.

Reply:
Witzig. Geht nur in Firefox und Safari: https://caniuse.com/mdn-css_properties_translate
Deswegen ist es mir auch nicht aufgefallen.

Reply:
Bei mir hatte es funktioniert. Aber transform:translate(...) scheint standardkonformer zu sein. Ich habe es korrigiert.